bigbluebutton-html5/imports/ui/components/layout/layout-manager/participantsChatOnlyLayout.jsx

```diff
@@ -0,0 +1,434 @@
+      width,
+      maxWidth,
+    };
+  };
+
+  const calculatesCameraDockBounds = () => {
+    const cameraDockBounds = {};
+
+    cameraDockBounds.top = 0;
+    cameraDockBounds.left = 0;
+    cameraDockBounds.right = 0;
+    cameraDockBounds.minWidth = 0;
+    cameraDockBounds.width = 0;
+    cameraDockBounds.maxWidth = 0;
+    cameraDockBounds.minHeight = 0;
+    cameraDockBounds.height = 0;
+    cameraDockBounds.maxHeight = 0;
+
+    return cameraDockBounds;
+  };
+
+  const calculatesMediaBounds = (mediaAreaBounds, sidebarSize) => {
+    const mediaBounds = {};
+    const { element: fullscreenElement } = fullscreen;
+
+    if (
+      fullscreenElement === 'Presentation'
+      || fullscreenElement === 'Screenshare'
+      || fullscreenElement === 'ExternalVideo'
+    ) {
+      mediaBounds.width = windowWidth();
+      mediaBounds.height = windowHeight();
+      mediaBounds.top = 0;
+      mediaBounds.left = !isRTL ? 0 : null;
+      mediaBounds.right = isRTL ? 0 : null;
+      mediaBounds.zIndex = 99;
+      return mediaBounds;
+    }
+
+    if (isMobile && cameraDockInput.numCameras > 0) {
+      mediaBounds.height = mediaAreaBounds.height * 0.7;
+    } else {
+      mediaBounds.height = mediaAreaBounds.height;
+    }
+    mediaBounds.width = mediaAreaBounds.width;
+    mediaBounds.top = DEFAULT_VALUES.navBarHeight + bannerAreaHeight();
+    mediaBounds.left = !isRTL ? mediaAreaBounds.left : null;
+    mediaBounds.right = isRTL ? sidebarSize : null;
+    mediaBounds.zIndex = 1;
+
+    return mediaBounds;
+  };
+
+  const calculatesLayout = () => {
+    const {
+      calculatesNavbarBounds,
+      calculatesActionbarBounds,
+      calculatesSidebarNavWidth,
+      calculatesSidebarNavBounds,
+      calculatesSidebarContentBounds,
+      isTablet,
+    } = props;
+    const { captionsMargin } = DEFAULT_VALUES;
+
+    const sidebarNavWidth = calculatesSidebarNavWidth();
+    const sidebarContentWidth = calculatesSidebarContentWidth(sidebarNavWidth.width);
+    const sidebarNavBounds = calculatesSidebarNavBounds();
+    const sidebarContentBounds = calculatesSidebarContentBounds(sidebarNavWidth.width);
+    const mediaAreaBounds = {
+      width: sidebarNavWidth.width + sidebarContentWidth.width,
+      left: 0,
+    };
+    const navbarBounds = calculatesNavbarBounds(mediaAreaBounds);
+    const actionbarBounds = calculatesActionbarBounds(mediaAreaBounds);
+    const sidebarNavHeight = calculatesSidebarNavHeight(navbarBounds.height,
+      actionbarBounds.height);
+    const sidebarSize = sidebarContentWidth.width + sidebarNavWidth.width;
+    const mediaBounds = calculatesMediaBounds(mediaAreaBounds, sidebarSize);
+    const sidebarContentHeight = calculatesSidebarContentHeight(navbarBounds.height,
+      actionbarBounds.height);
+    const cameraDockBounds = calculatesCameraDockBounds(
+      mediaBounds,
+      mediaAreaBounds,
+      sidebarNavWidth.width,
+      sidebarContentWidth.width,
+      sidebarContentHeight.height,
+    );
```

Comment:
This function expects no arguments

```suggestion
    const cameraDockBounds = calculatesCameraDockBounds();
```